tests/PS01/readme.md

```diff
@@ -80,11 +80,14 @@ Not covered by the IDS file (must be checked otherwise):
         - one `.LENGTHUNIT.` and
         - one `.PLANEANGLEUNIT.`.
 - (RI-03) An `IfcGeometricRepresentationContext` shall be associated with the `IfcProject`. The project origin should be `(0,0,0)`.
+- (RI-64) The placement of `IfcAnnotation` shall be relative to the `IfcFacility` in which it is contained.
```

Comment:
Same as bullet 2 here https://github.com/buildingSMART/IFC4.x-IF/pull/48#discussion_r1260868058

Reply:
Resolved in 90c60c1